handbook/design/index.md

```diff
@@ -10,6 +10,22 @@ Here are a few of the things we strive to make true:
 - Communicate openly, and solicit feedback frequently
 - To be egoless when soliciting responses to our work
 
+## How to work with design
+
+*This section is WIP*
+
+Advice on how to best work with the design team at Sourcegraph. 
+
+We wanted to document this as design is new to Sourcegraph, and it might not be immediately clear how to get the most out of the design team. 
+
+- If you're in a pinch, the design team is here to help, so don't hesitate to solicit feedback from us. Even if you think your problem might not pertain to design, don't leave it to chance
```

Comment:
This is really unclear what is work not pertaining to design - would you want to be part of an API design conversation for example? What sort of pinch would they be in? I think being more direct is sufficient.

```suggestion
- The design team is here to help, so don't hesitate to solicit feedback from us on your work.
```